### PR TITLE
EameHr58: Align disconnected hint div with others on page

### PR DESCRIPTION
**Pull request by bjgill**
Line up the box around the disconnection hint with the boxes below around the other IDPs for signin.

### CODE REVIEW

**Comment by bjgill** [comment]
What this looks like:

screen shot 2019-03-07 at 10 59 24

**Comment by bjgill** [comment]
@jakubmiarka - this fixes the wonkiness idendified in https://github.com/alphagov/verify-frontend/pull/667#pullrequestreview-208628484

File: app/assets/stylesheets/pages/_company.scss [hunk 1]
```diff
@@ -18,6 +18,10 @@
   }
 }
 
+.company-full-width {
+  padding: 0px 15px;
```
Comment (galund) [inline app/assets/stylesheets/pages/_company.scss]: Zeroes should not have units

**Comment by jakubmiarka** [comment]
Should we also fix the horizontal scrolling at the bottom? Where the list of disconnected IDP is? On mobile I mean.

**Comment by bjgill** [comment]
@jakubmiarka - thanks for the comment. I've now fixed the scrolling problem you identified.

screen shot 2019-03-07 at 11 35 04

**Pull request merged**